app/models/report.rb

```diff
@@ -1,7 +1,43 @@
 # frozen_string_literal: true
 
 class Report < ApplicationRecord
+  after_commit :push_to_slack, on: :create
+
   belongs_to :user
 
   validates :task_today, :obstacles_today, presence: true
+
+  def push_to_slack
+    thread_ts = SlackThread.find_by(created_on: Time.zone.today).thread_ts
+
+    uri = URI.parse('https://slack.com/api/chat.postMessage')
+
+    req_options = { use_ssl: uri.scheme == 'https' }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/reportedly-api/blob/fb89d1c90ff855d229a9544a010555f929601d1c/app/models/report.rb#L15"></span>Refers to 'uri' more than self (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>